public/app/core/components/Signup/SignupPage.tsx

```diff
@@ -80,16 +81,17 @@ export const SignupPage: FC<Props> = (props) => {
               </Field>
               {!getConfig().autoAssignOrg && (
                 <Field label="Org. name">
-                  <Input {...register('orgName')} placeholder="Org. name" />
+                  <Input id="org_name" {...register('orgName')} placeholder="Org. name" />
```

Comment:
```suggestion
                  <Input id="org-name" {...register('orgName')} placeholder="Org. name" />
```
I think we'd have a consistent naming pattern for ids.

Reply:
addressed